python-main.js

```diff
@@ -1289,6 +1287,12 @@ function web_editor(config) {
         REPL.prefs_.set('font-size', getFontSize());
     }
 
+    function openModalMsg(title,content){
+        $("#modal-msg-overlay-container").css("display", "flex");
+        $("#modal-msg-title").text(title);
+        $("#modal-msg-content").html(content);       
+        
+    }
```

Comment:
Can we remove the empty line before the closing bracket and add an empty line in-between the functions?